modules/SofaMiscMapping/CenterOfMassMapping.inl

```diff
@@ -78,7 +85,7 @@ void CenterOfMassMapping<TIn, TOut>::apply( const sofa::core::MechanicalParams*
     //with Xi: position of the dof i, Mi: mass of the dof i, and Mt : total mass of the object
     for (unsigned int i=0 ; i<parentPositions.size() ; i++)
     {
-        outX += parentPositions[i].getCenter() * masses->getElementMass(i);
+        outX += parentPositions[i].getCenter() * l_mass->getElementMass(i);
```

Comment:
Get the pointer once.

Reply:
Sure, I can do that. Wouldn't that kind of de-referencing be automatically optimized by the compiler though?